profile-widget/diveprofileitem.h

```diff
@@ -20,15 +18,10 @@
 class DivePlotDataModel;
 class DiveTextItem;
 class DiveCartesianAxis;
-class QAbstractTableModel;
 struct plot_data;
 struct dive;
 
-class AbstractProfilePolygonItem : public QObject, public QGraphicsPolygonItem {
-	Q_OBJECT
-	Q_PROPERTY(QPointF pos WRITE setPos READ pos)
-	Q_PROPERTY(qreal x WRITE setX READ x)
-	Q_PROPERTY(qreal y WRITE setY READ y)
```

Comment:
I think we used this from QML at some point... but I can't remember for what - and I don't thing we still do? At least I couldn't find this via a few careful `git grep` commands... the sad thing is that we don't get build time warnings if we break this...

Reply:
This was needed for the `QPropertyAnimation`. Since the animation is now routed via a central point, it is not needed anymore.